playbooks/openstack/configuration.md

```diff
@@ -1148,3 +1149,155 @@ If you are running a template router to expose your statistics, there are a few
     port_range_max: 1936
 ```
 
+## Using CRI-O
+There are some different scenarios to customize the container runtime in the
+instances:
+
+* All hosts use docker (no changes required)
+* All hosts using cri-o. Modify the all.yml and add the following variables:
+
+```
+openshift_use_crio_only: true
+openshift_use_crio: true
+openshift_openstack_master_group_name: node-config-master-crio
+openshift_openstack_infra_group_name: node-config-infra-crio
+openshift_openstack_compute_group_name: node-config-compute-crio
+# cockpit-docker is installed if using cockpit docker as dependency
+# setting osm_use_cockpit=false to avoid that
+osm_use_cockpit=false
```

Comment:
Do the openshift_* and osm_use_cockpit vars belong in OSEv3.yml instead of all.yml?

Reply:
At the end it will be the same, as OSEv3 contains nodes, masters, etc. and all contains all groups, but you are right, to be 100% accurate, it should be in the OSEv3.yml file :) I'll fix it.

Reply:
Fixed. Do you mind to take a look? Thanks!

Reply:
Thanks for the change! One clarification: we like to put the openshift_openstack_* variables in all.yml - variables which define how the OpenStack resources would be provisioned; while putting the others in OSEv3.yml. So in this case the openshift_openstack_*_group_name variables should be set in all.yml, while the others are set in OSEv3.yml.

Reply:
Done!

Reply:
Looks good, thanks for the changes!